tests/test_bench.py

```diff
@@ -370,28 +307,12 @@ def test_dj_constant_oracle() -> None:
     assert qc.depth() > 0
 
 
-def test_unidirectional_coupling_map() -> None:
-    """Test the unidirectional coupling map for the OQC Lucy device."""
-    qc = get_benchmark(
-        benchmark_name="dj",
-        level="mapped",
-        circuit_size=3,
-        compiler="tket",
-        provider_name="oqc",
-        device_name="oqc_lucy",
-    )
-    # check that all gates in the circuit are in the coupling map
-    cmap = [(c[0], c[1]) for c in OQCProvider.get_device("oqc_lucy").coupling_map]
-    assert qc.valid_connectivity(arch=pytket.architecture.Architecture(cmap), directed=True)
-
-
 @pytest.mark.parametrize(
```

Comment:
I could imagine that we can remove some of the tested benchmark instances. I used so many to cover all different combinations. Since tket is not available anymore, we most likely can reduce the number of test instances.